10_code/40-concatenation_death_data.py

```diff
@@ -0,0 +1,34 @@
+# %%
```

Comment:
It appears comments like "# %%" are a result of converting .ipynb file to .py. We can probably remove that for cleaner code.

Reply:
good point. got that removed